digital-workspace/latest/governance/folder-rules.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Folder rules
+---
+
+In the Digital Workspace you can define folder rules to manage your content automatically, in the same way email clients use rules to filter messages and incoming spam. Rules dictate how content entering, leaving, or currently residing in a folder is managed. When you define a rule, it only applies to new content added to the folder. Items that were in the folder before the rule was defined aren't affected by it.
```

Comment:
Rules dictate how content entering, leaving, or currently residing in a folder is managed - needs rewrite

Reply:
Hi @lilustig can you be a little more specific please? 
That sentence was taken from here https://docs.alfresco.com/content-services/latest/using/content/rules/

Reply:
Lilian has said to ignore her comment and publish.